src/main/resources/swagger/api-docs.yaml

```diff
@@ -200,6 +215,41 @@ paths:
             - openid
             - email
             - profile
```

Comment:
No change to `get` below...just moved it below `put` so the two versions of `put` would be stacked.